types/webpack.config.d.ts

```diff
@@ -0,0 +1,91 @@
+declare const _exports: ({
```

Comment:
Should the jest.config.d.ts + webpack.config.d.ts files be included?

Reply:
Removed